geniza/corpus/models.py

```diff
@@ -486,12 +486,15 @@ def metadata_prefetch(self):
             .prefetch_related(
                 "tags",
                 "languages",
-                Prefetch(
-                    "textblock_set",
-                    queryset=TextBlock.objects.select_related(
-                        "fragment", "fragment__collection"
-                    ),
-                ),
+                # @TODO: This is causing error?
+                #       ValueError: 'textblock_set' lookup was already seen with a different queryset. You may need to adjust the ordering of your lookups.
+                #
+                # Prefetch(
+                #     "textblock_set",
+                #     queryset=TextBlock.objects.select_related(
+                #         "fragment", "fragment__collection"
+                #     ),
+                # ),
```

Comment:
ah, now that I see it in context maybe I know what is happening here — is it possible that there's some context in which you're calling the prefetch twice?

Reply:
maybe admin queryset + export — are we calling the prefetch even if the queryset is passed in?

Reply:
Perhaps it's this line in admin.py:

```python
def get_queryset(self, request):
        return super().get_queryset(request).metadata_prefetch()
```

Reply:
I believe it is. This solves the tests and works on admin and CLI:
```python
def get_queryset(self, request):
        return super().get_queryset(request) #.metadata_prefetch()
```